docs/technical_reference/unit_models/reverse_osmosis_0D.rst

```diff
@@ -44,7 +44,7 @@ Sets
    "Phases", ":math:`p`", "['Liq']"
    "Components", ":math:`j`", "['H2O', 'NaCl']*"
 
-*Solute depends on the imported property model; example shown here is for the NaCl property model.
+*Solute depends on the imported property model; example shown here is for the NaCl property model.*
```

Comment:
Rather than setting as bold text, this was meant to be a footnote for line 45: `"['H2O', 'NaCl']*"` Thus, I think we should keep the original line intact.

Reply:
I think it should read this then; the existing was causing a warning upon build.
```suggestion
\*Solute depends on the imported property model; example shown here is for the NaCl property model.
```